services/mentorship/Api/createMentorship.ts

```diff
@@ -70,7 +70,7 @@ const createMentorshipAPI: APIGatewayProxyHandler = async (event) => {
 
   try {
     const {
-      Item: { date, is_occupied },
+      Item: { date, is_occupied, duration },
```

Comment:
If `Item` is undefined this would throw an error that its hard to catch.

Although in this case I don't think this would cause any problems I'm not a great fan of destructuring in this way.

Reply:
Yes, you right, i'll change it. Thanks